src/aktualizr_lite/test_lite.sh

```diff
@@ -49,10 +48,16 @@ add_target foo2
 akrepo --command signtargets
 
 pushd $dest_dir
-python3 -m http.server $port &
+python3 -m http.server 0&
 pid=$!
-
-# curl http://localhost:$port/repo/image/targets.json | json_pp
+port=
+try=0
+while [[ -z $port  && $try -le 100 ]]; do
+    port=$(netstat -lpan 2> /dev/null | awk -F '[ :]+' "/$pid\\/python3/ {print \$5}")
+    sleep 0.2
+    try=$((try+1))
+done
+echo "http server listening on $port"
```

Comment:
Would it be better to move this port-finding logic into a reusable script?

Reply:
yes that's fair, I'll try something